examples/chip-tool-darwin/templates/tests/partials/test_cluster.zapt

```diff
@@ -126,18 +126,13 @@ class {{filename}}: public TestCommandBridge
     {{#*inline "testCommand"}}Test{{asUpperCamelCase label}}_{{index}}{{/inline}}
     CHIP_ERROR {{>testCommand}}()
     {
-        SetIdentity("{{identity}}");
```

Comment:
I did replaced `GetConnectedDevice` by `GetDevice(const char * _Nullable identity)`. There is still some code paths that calls `SetIdentity` because the pairing delegate does not allow to retrieve the initial context (which is unpractical..).